### PR TITLE
issue/1214-move-logged-in-user

### PR DESCRIPTION
**Pull request by Goff-Davis**
- Adjustments to logged in user welcome message and avatar to work with changes to css

Depends on [this](https://github.com/ucfopen/Materia-Server-Client-Assets/pull/65) pull request in Server Client Assets.

### CODE REVIEW

File: fuel/app/themes/default/partials/header.php [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?php // @codingStandardsIgnoreStart ?>
 <header ng-controller="currentUserCtrl" class="{loggedIn: currentUser.loggedIn==true}" >
+	<!-- <meta content="width=device-width, initial-scale=1" name="viewport"> -->
```
Comment (FrenjaminBanklin) [inline fuel/app/themes/default/partials/header.php]: Toss this if we don't need it.

**Comment by iturgeon** [comment]
Make sure you test a super long name like "Frieda Robscheit-Robbins"

**Comment by Goff-Davis** [comment]
A couple of things. I matched the image above, and added a border-radius to the user image. That can be removed but I thought that it looked better, and matched the mobile burger menu. I made the logout text smaller than the user's name text like in the image but I think that that looks a bit too small.

**Comment by Goff-Davis** [comment]
This breaks the log in functionality, don't merge it until I have it fixed.

**Comment by Goff-Davis** [comment]
The failing check shouldn't be related. You can rerun it until it works if you want to merge this in.

**Comment by FrenjaminBanklin** [comment]
test please

**Pull request merged**